lang/cs/Org.Apache.REEF.Tests/Functional/IMRU/TestFailMapperTasksOnDispose.cs

```diff
@@ -62,7 +62,8 @@ public override void TestFailedMapperOnLocalRuntime()
             // No failed evaluators or tasks.
             Assert.Equal(0, failedEvaluatorCount);
             Assert.Equal(0, failedTaskCount);
-            Assert.Equal(numTasks, completedTaskCount);
+            Assert.True(numTasks >= completedTaskCount);
+            Assert.True(completedTaskCount > 1);
```

Comment:
Why is this condition strictly greater? I think it's not impossible that master task will be the first one to complete